charts/bungee/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+name: bungee
+version: 1
+appVersion: SeeValues
+description: "Minecraft proxy server (BungeeCord, Waterfall, Velocity)"
+keywords:
+  - bungee
+  - game
+  - server
+home: "https://github.com/ArchitectSMP/charts"
+sources:
+  - "https://github.com/ArchitectSMP/charts"
```

Comment:
in case, could keep yours listed also...but might be confusing

Reply:
No no, ours will go away once this is merged